draft-irtf-cfrg-fiat-shamir.md

```diff
@@ -54,6 +54,20 @@ It depends on:
 - A _hash function_ implementing the duplex sponge interface, capable of absorbing inputs incrementally and squeezing variable-length unpredictable messages.
 - A _codec_, which securely remaps prover elements into the base alphabet, and outputs of the duplex sponge into verifier messages (preserving the distribution).
 
+# Security Considerations
+
+We focus on the security guarantees of the non-interactive Fiat-Shamir transformation, where they provide the following guarantees (in the random oracle model) for Non-interactive Sigma Protocols:
+
+- **Knowledge soundness**: If the proof is valid, the prover must have knowledge of a secret witness satisfying the proof statement. This property ensures that valid proofs cannot be generated without possession of the corresponding witness.
```

Comment:
"we preserve the soundness of the interactive protocol" / "carries over the properties"